pkgs/top-level/aliases.nix

```diff
@@ -673,6 +673,7 @@ mapAliases ({
   libcxxabi = throw "'libcxxabi' was merged into 'libcxx'"; # Converted to throw 2024-03-08
   libdwarf_20210528 = throw "'libdwarf_20210528' has been removed because it is not used in nixpkgs, move to libdwarf"; # Added 2024-03-23
   libgme = game-music-emu; # Added 2022-07-20
+  libgnome-keyring3 = libgnome-keyring; # Added 2024-06-22
```

Comment:
Maybe this should be part of the previous commit.

Reply:
I think a separate commit would be cleanest but I just cannot be bothered.